src/sagemaker/git_utils.py

```diff
@@ -72,44 +81,182 @@ def git_clone_repo(git_config, entry_point, source_dir=None, dependencies=None):
             updated_paths["entry_point"] = os.path.join(repo_dir, entry_point)
         else:
             raise ValueError("Entry point does not exist in the repo.")
-
-    updated_paths["dependencies"] = []
-    for path in dependencies:
-        if os.path.exists(os.path.join(repo_dir, path)):
-            updated_paths["dependencies"].append(os.path.join(repo_dir, path))
-        else:
-            raise ValueError("Dependency {} does not exist in the repo.".format(path))
+    if dependencies is None:
+        updated_paths["dependencies"] = None
```

Comment:
Do you need to update the ["dependencies"] if it's None? I thought you already assigned the values here: https://github.com/aws/sagemaker-python-sdk/blob/ed74c76711086202151ec35b0e0dc04172f85f4a/src/sagemaker/git_utils.py#L69

Reply:
Yes, you are right.